packages/vue/src/components/organisms/SfHeader/_internal/SfHeaderNavigationItem.vue

```diff
@@ -1,10 +1,40 @@
 <template>
   <div class="sf-header-navigation-item" v-on="$listeners">
-    <slot />
+    <div class="sf-header-navigation-item--desktop">
+      <SfLink class="sf-header-navigation-item__link" :link="link">{{
+        label
```

Comment:
Another thing I'm wondering about. Is it necessary to have links here? There are more like buttons reacting on hover. I'm talking about the case in the story when after clicking on items there I receive "Cannot GET /clothing" error. Maybe it could be changed only in the story, wdyt?

Reply:
You are completely right, I removed it from story :)